templates/stepback-qa-prompting/pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [tool.poetry]
-name = "stepback_qa_prompting"
+name = "stepback-qa-prompting"
```

Comment:
fyi - this doesnt matter

Reply:
infact, this is actually NOT desirable (the `new` command creates it like this, so we should be consistent)